### DISCUSSION TITLE
How to prevent locking files when loading usd stage

### DISCUSSION THREAD

**Discussion by jhultgre** [Autodesk/maya-usd#3760] [Q&A]
When loading in a stage maya will hold on to a file handle for the usd files making up the stage.
This is causing issues since other processes aren't able to modify the files, such as syncing a newer version of a usd asset from perforce.

Is there a way tell maya to release the files after it has loaded in the stage?

**Comment by wallworm** [comment]
Thanks for bringing this up. I'll ask the team about it.
